addons/web/tooling/_eslintrc.json

```diff
@@ -1,7 +1,8 @@
 {
   "extends": ["eslint:recommended", "plugin:prettier/recommended"],
   "parserOptions": {
-    "sourceType": "module"
+    "sourceType": "module",
+    "ecmaVersion": 2022
```

Comment:
Is this an intended change (which should be in this pr)? Just asking

Reply:
It is an intended change yes, it allows eslint to parse class properties (eg  `static template = "pos.SomeTemplate";`), may be should be in a separate commit but I didn't want to create a separate PR for one line of diff :sweat_smile:

Reply:
Yeah, it's okay, just wanted to be sure if this was intentional or a mistake